future.tex

```diff
@@ -108,6 +33,38 @@ \subsection{Summary}
 	}
 \end{table}
 
+The next table lists different nations in the EU and their nuclear program plans.
```

Comment:
Never say "The next table" because the tables can move around in relation to the text when the document is built, or later when you add some other table in between and forget to go back and edit. Instead, say `The planned nuclear deployment strategies of various EU nations is summarized in Table \ref{tab:eu_deployment}`. THEN, use the fabulous quality of latex and label your table in the table environment with `\label{tab:eu_deployment}`